src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt

```diff
@@ -1490,7 +1490,6 @@ TypesMustExist : Type 'System.Data.SqlClient.SqlColumnEncryptionCertificateStore
 TypesMustExist : Type 'System.Data.SqlClient.SqlColumnEncryptionCngProvider' does not exist in the implementation but it does exist in the contract.
 TypesMustExist : Type 'System.Data.SqlClient.SqlColumnEncryptionCspProvider' does not exist in the implementation but it does exist in the contract.
 TypesMustExist : Type 'System.Data.SqlClient.SqlColumnEncryptionKeyStoreProvider' does not exist in the implementation but it does exist in the contract.
-CannotRemoveBaseTypeOrInterface : Type 'System.Data.SqlClient.SqlCommand' does not implement interface 'System.ICloneable' in the implementation but it does in the contract.
```

Comment:
This is created/updated automatically when running build.cmd and getting a successful build. This is a diff in APIs we have in between netcoreapp and desktop. So don't worry about this. You can either remove this change from your PR or leave it, sooner or later it will get updated.